base/mpfr.jl

```diff
@@ -1054,11 +1054,15 @@ Set the precision (in bits, by default) to be used for `T` arithmetic.
 If `base` is specified, then the precision is the minimum required to give
 at least `precision` digits in the given `base`.
 
-!!! warning
-
-    This function is not thread-safe. It will affect code running on all threads, but
-    its behavior is undefined if called concurrently with computations that use the
-    setting.
+!!! note
+    Temporarily sets the precision for BigFloat operations to `precision` bits within the
+    scope of function `f`. This function is now thread-safe due to the use of `ScopedValue`,
```

Comment:
In the context of a docstring, "now" isn't useful, it doesn't give a temporal or version reference. Perhaps you meant to add a "compat" admonition as done below?